gamemodes/terrortown/gamemode/shared/hud_elements/tttsidebar/pure_skin_sidebar.lua

```diff
@@ -92,31 +94,65 @@ if CLIENT then
 		self:DrawLines(pos.x, curY, size.w, size.w, item.hud_color.a * factor)
 
 		if isfunction(item.DrawInfo) then
-			local info = item:DrawInfo()
+			local info = TryT(item:DrawInfo())
 			if info then
-				-- right bottom corner
-				local tx = pos.x + size.w - 5
-				local ty = curY +  size.w - 1
-				local pad = 5 * self.scale
-
-				surface.SetFont("PureSkinItemInfo")
-
-				local infoW, infoH = surface.GetTextSize(info)
+				local infoW, infoH = draw.GetTextSize(info, "PureSkinItemInfo")
 				infoW = infoW * self.scale
-				infoH = (infoH + 2) * self.scale
+				infoH = (infoH + 3) * self.scale
+
+				-- right bottom corner
+				local pad = 4 * self.scale
+				local tx = pos.x + size.w - 1 * self.scale - 0.5 * infoW
+				local ty = curY + size.w + 4 * self.scale - 0.5 * infoH
 
-				local bx = tx - infoW * 0.5 - pad
+				local bx = tx - 0.5 * infoW - pad
 				local by = ty - infoH * 0.5
 				local bw = infoW + pad * 2
 
 				self:DrawBg(bx, by, bw, infoH, item.hud_color)
 
-				draw.AdvancedText(info, "PureSkinItemInfo", tx, ty, fontColor, TEXT_ALIGN_CENTER, TEXT_ALIGN_CENTER, false, self.scale)
+				draw.AdvancedText(info, "PureSkinItemInfo", tx, ty, fontColor, TEXT_ALIGN_CENTER, TEXT_ALIGN_CENTER, true, self.scale)
 
 				self:DrawLines(bx, by, bw, infoH, item.hud_color.a)
 			end
 		end
 
+		if GAMEMODE.ShowScoreboard then
+			local xText = pos.x + size.w + self.padding
+			local offsetTitle = -2 * self.scale
+			local offsetLines = {19 * self.scale, 33 * self.scale}
+
+			local name = item.EquipMenuData and item.EquipMenuData.name or (item.name or "")
+
+			-- alow dynamic status names
+			if istable(name) then
+				name = name[item.active_icon]
```

Comment:
Why is the active icon here used to index the name? Like why is it the icon? This needs comments or a renaming

Reply:
this is the horrible status system (blame on me). I can't really change the name. But the active icon ID is used to select the active icon. We could rework the STATUS system, but not in this PR

Reply:
okay, i got confused by the whole naming here, i did not expect item to also be status effects. This is really confusing